src/utils/css.ts

```diff
@@ -43,9 +52,9 @@ export const Focus = styled.div<any>`
       opacity: 0;
       transition: 150ms ease-in-out;
 
-      ${parent}:focus > &,
-      ${parent}:focus ~ &,
-      ${parent}:focus ~ div > & {
+      ${parent}:${pseudoSelector} > &,
```

Comment:
Were you not getting a `Cannot parse selector   parseError` for this?

Reply:
Um I did a no verify.